filament/include/filament/SwapChain.h

```diff
@@ -143,6 +143,7 @@ namespace filament {
 class UTILS_PUBLIC SwapChain : public FilamentAPI {
 public:
     static const uint64_t CONFIG_TRANSPARENT = driver::SWAP_CHAIN_CONFIG_TRANSPARENT;
+    static const uint64_t CONFIG_READABLE = driver::SWAP_CHAIN_CONFIG_READABLE;
```

Comment:
There should be some documentation for this new flag.

Reply:
done.